src/plugins/email/email.c

```diff
@@ -0,0 +1,85 @@
+/**
+ * @file
+ *
+ * @brief Source for email plugin
+ *
+ * @copyright BSD License (see doc/LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <kdberrors.h>
+#include <regex.h>
+#include <stdio.h>
+
+#include "email.h"
+
+static int validateEmail (Key * key, Key * parentKey)
+{
+	const char * addr = keyString (key);
```

Comment:
metadata of key needs to be checked: is this value supposed to be an email address?